### PR TITLE
adding Georgia Tech project

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

**Comment by brianhlin** [comment]
Should we use the full organizational name, `Georgia Institute of Technology`?

**Comment by matyasselmeci** [comment]
That's the one that's in there, so yes.

File: projects/GATech_Chau.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name: GATech_Chau
+Description: Attention Shift Visualization
+Department: Computer Science and Engineering
+FieldOfScience: Computer Science
+Organization: Georgia Tech
```
Comment (matyasselmeci) [inline projects/GATech_Chau.yaml]:
```suggestion
Organization: Georgia Institute of Technology
```

**Comment by ChristinaLK** [comment]
Is there an easy way to get a master list of all the institutions? I feel like I do this every time I add a new project.

**Comment by matyasselmeci** [comment]
Let me add a script.

**Pull request merged**